tools_iuc.yaml

```diff
@@ -3272,6 +3290,15 @@ tools:
   - name: sina
     owner: iuc
     tool_panel_section_label: Multiple Alignments
+  
+  - name: emboss_needle
```

Comment:
I think this is already included. PLease check that you didn't add duplicates ...